src/SonarScanner.MSBuild.PreProcessor/ProcessedArgs.cs

```diff
@@ -217,13 +226,13 @@ public string GetSetting(string key, string defaultValue)
             properties.TryGetProperty(SonarProperties.OperatingSystem, out var operatingSystem)
                 ? operatingSystem.Value
                 : operatingSystemProvider.OperatingSystem() switch
-                  {
-                      PlatformOS.Windows => "windows",
-                      PlatformOS.MacOSX => "macos",
-                      PlatformOS.Alpine => "alpine",
-                      PlatformOS.Linux => "linux",
-                      _ => "unknown"
-                  };
+                {
+                    PlatformOS.Windows => "windows",
+                    PlatformOS.MacOSX => "macos",
+                    PlatformOS.Alpine => "alpine",
+                    PlatformOS.Linux => "linux",
+                    _ => "unknown"
+                };
```

Comment:
I think the previous alignment was better, the block is conceptually linked to the switch and not to the ternary expression.

Reply:
VS auto-formatting doesn't like this indentation.

Reply:
I can bring the same argument for Rider

Reply:
*that does not like the new identation